vrrpd/vrrp_northbound.c

```diff
@@ -40,12 +40,25 @@ static int lib_interface_vrrp_vrrp_group_create(struct nb_cb_create_args *args)
 	uint8_t version = 3;
 	struct vrrp_vrouter *vr;
 
-	if (args->event != NB_EV_APPLY)
-		return NB_OK;
-
-	ifp = nb_running_get_entry(args->dnode, NULL, true);
+	ifp = nb_running_get_entry(args->dnode, NULL, false);
 	vrid = yang_dnode_get_uint8(args->dnode, "./virtual-router-id");
 	version = yang_dnode_get_enum(args->dnode, "./version");
+
+	switch (args->event) {
+	case NB_EV_VALIDATE:
+		if (ifp) {
+			vr = vrrp_lookup(ifp, vrid);
+			if (vr && vr->autoconf)
+				return NB_ERR_VALIDATION;
```

Comment:
nit: it's always good to write something to `args->errmsg` to inform the user exactly why the validation failed.

Reply:
Ack, I'll do that